_templates/api-story-card/with-prompt/src/components/App/index-import.ejs.t

```diff
@@ -0,0 +1,7 @@
+---
+to: packages/<%=package%>/src/components/App/index.js
+inject: true
+after: \nimport(?![\s\S]*\nimport).*
```

Comment:
This regex will inject this after the the last occurrence of a new line that begins with "import" (i.e. right after the other import statements) I tried to add a comment in here clarifying that, but I could not figure out how to add comments to these files without breaking something...

Reply:
I think [this solution may work](https://github.com/tj/ejs/issues/76#issuecomment-14104705)